common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -142,12 +142,13 @@ def test_student_view_init(self):
         seq_module = SequenceModule(runtime=Mock(position=2), descriptor=Mock(), scope_ids=Mock())
         self.assertEqual(seq_module.position, 2)  # matches position set in the runtime
 
+    @patch('xmodule.seq_module.SequenceModule._get_user', return_value=UserFactory.build())
```

Comment:
not sure if i should be doing something different but I did this last time I was in this same code

Reply:
Huh. I'm surprised you have to do anything and Django doesn't just handle calls to get_user_model gracefully.

Separately, is it possible to mock this at the class level? I'm honestly not sure, but would get to save you from a lot of these other changes.

Reply:
i'm getting the django user to pass to the content type gating checks
when looking at this I was mocking this method because the user wasn't actually created, or if i get the Xblock user it doesn't have an id for the content type gating check to grab (the get_user_model part of it works fine) open to not mocking the function but had trouble figuring out how i'd make that work

i can mock it at a class level, but that would require adding a mocked_user parameter to every function in the class, whereas this way I only had to add it to a subset of the functions that were affected

Reply:
Nah, I was just surmising. What you have is great.